cfme/configure/configuration/__init__.py

```diff
@@ -423,8 +428,10 @@ class ServerLogDepot(Pretty):
 
     """
     elements = Region(
-        locators={},
-        infoblock_type="form"
+        locators={
+            "last_message": InfoBlock.create("Basic Info", "Last Message"),
+            "last_log_collection": InfoBlock.create("Basic Info", "Last Log Collection"),
+        },
```

Comment:
You mean something like this?
```python
informations = InfoBlock("Informations")

informations.uptime == "xyz"
```
?

This would however require some translation dictionary for the child keys which leads to same or worse typing length IMO.

btw. everything is cached so this is just a matter of typing.